atf_eregs/static/regulations/css/scss/module/custom/_about.scss

```diff
@@ -24,6 +24,19 @@
     background: $atf_carrots;
 }
 
+.about-stopcompare {
+    margin-top: 80px;
+    margin-left: 20px;
+}
+
+.icon-red {
+    color: #d14124;
+}
+
+img.about-stopcompare-img {
```

Comment:
NBD, but per https://pages.18f.gov/frontend/#css-specificity, this shouldn't include the `img`